src/System.Memory/ref/System.Memory.cs

```diff
@@ -246,16 +245,17 @@ public static partial class SequenceReaderExtensions
     }
     public ref partial struct SequenceReader<T> where T : unmanaged, System.IEquatable<T>
     {
+        private readonly object _dummy;
```

Comment:
This should also have a dummy field for a value type, right?

Reply:
Since `SequenceReader<T>` contains a `SequencePosition` fields, which contain a reference type as a field (`object`), I believe the dummy field should be reference type.

At least according to:
https://github.com/dotnet/arcade/blob/61cc7d4d347300835925ce5245bfc3ecb6b621d7/src/Microsoft.Cci.Extensions/Writers/CSharp/CSharpWriter.cs#L153-L161

cc @ericstj

Reply:
I ran the tool again and it didn't add anything.

Reply:
Manually put it in